yazi-config/preset/theme-dark.toml

```diff
@@ -128,6 +128,18 @@ btn_labels = [ "  [Y]es  ", "  (N)o  " ]
 # : }}}
 
 
+# : Spot {{{
+
+[spot]
+border   = { fg = "blue" }
+title    = { fg = "blue" }
+headers  = { fg = "green" }
+keys     = {}
```

Comment:
This seems not used, please remove it

Reply:
Sorry, I mean `keys` - for some reason GitHub showed them all these properties

Reply:
My idea for `keys` is to use them to change the color of the left column while the `values` will control the right column color.

Reply:
The issue here is that Yazi doesn't know what counts as `keys`, because the spotter can provide [any renderable element](https://yazi-rs.github.io/docs/plugins/layout). 

Currently, we first implemented it for tables, which only have [the concepts of rows, columns, and cells](https://docs.rs/ratatui/latest/ratatui/widgets/struct.Table.html). This way will allow for more advanced rendering needs in the future, such as rendering a paragraph or a list, not just a table.

Reply:
Mhm, let me experiment when I get access to `THEME` in Lua plugins then.